bench/.editorconfig

```diff
@@ -1669,6 +1718,74 @@ dotnet_diagnostic.CA5404.severity = none
 # Help Link: https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca5405
 dotnet_diagnostic.CA5405.severity = none
 
+# Title    : Use source generated logging methods for improved performance
+# Category : Performance
+dotnet_diagnostic.EA0000.severity = warning
```

Comment:
Nothing against the name, but I'm just curious. Why EA? What does that stand for?

Reply:
Oh, looks like ExtraAnalyzers?

Reply:
Yeah.